openmdao/surrogate_models/kriging.py

```diff
@@ -150,6 +157,19 @@ def _calculate_reduced_likelihood_params(self, thetas=None):
         params['U'] = U
         params['Vh'] = Vh
 
+        # Newly added for EGOLF
+        one = np.ones([self.n_samples,1])
+        R_inv = Vh.T.dot(np.dot(np.diag(inv_factors),U.T))
+        mu = (one.T.dot(np.dot(R_inv,Y)))/(one.T.dot(np.dot(R_inv,one)))
+        SigmaSqr = (Y-one*mu).T.dot(np.dot(R_inv,(Y-one*mu)))/self.n_samples
+        c_r = np.dot(R_inv,(Y - one*mu))
```

Comment:
I see you are using a reduced likelihood function. Could you please send me the reference to the reduced function?
Thanks.